pom.xml

```diff
@@ -1,128 +1,196 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
This file has changed from two spaces indent to four spaces indent, which makes it impossible to review the actual changes. If the code style should change, I'd recommend doing this in a separate PR.